examples/combined-example.py

```diff
@@ -0,0 +1,174 @@
+"""
+Example of how to render lists, tables and trees.
+"""
+import random
+
+from observ import reactive
+import pygfx as gfx
+from PySide6 import QtWidgets
+from wgpu.gui.qt import WgpuCanvas
+
+from collagraph import Collagraph, create_element as h, EventLoopType
+from collagraph.components import Component
+from collagraph.renderers import PygfxRenderer, PySideRenderer
```

Comment:
I guess I would recommend adjusting your `__init__.py`  a little bit just so everything useful is exposed directly from the top-level package. That way user-code won't depend on the package's structure:

```python
from collagraph import (
    Collagraph,
    create_element as h,
    EventLoopType,
    Component,
    PygfxRenderer,
    PySideRenderer,
)
```

I would also recommend exporting `create_element` as `h` as well. You can just add `h = create_element` to `__init__.py`, so that the alias is available by default:

```python
from collagraph import Collagraph, h, EventLoopType, Component, PygfxRenderer, PySideRenderer
```

---

Aside, personally I would prefer to import like this, to be able to clearly namespace things as coming from collagraph:

```python
import collagraph as cg
from collagraph import h  # or h = cg.h
```

Allows things like this:

```python
class RenderWidget(cg.Component):
```

And still write render functions like this:

```py
    def render(self):
        return h("WgpuCanvas", {"minimum_height": 400, "minimum_width": 600})
```